cpp/include/raft/linalg/detail/matrix_vector_op.cuh

```diff
@@ -22,80 +22,10 @@ namespace raft {
 namespace linalg {
 namespace detail {
 
-namespace {
```

Comment:
It won't break cuML because all template types are inferred.
It could in theory break other projects if they provide the template list explicitly, but I'm not aware of any.